airbyte-cdk/bulk/core/extract/src/test/kotlin/io/airbyte/cdk/read/RootReaderIntegrationTest.kt

```diff
@@ -335,7 +335,7 @@ data class TestCase(
             )
             for (actualState in actual!!) {
                 Assertions.assertTrue(
-                    actualState in expected,
+                    actualState.toString() in expected.map { it.toString() },
```

Comment:
stringification is required because of the deep-copying added to `BufferingOutputConsumer`, this causes the hash values to change or something, failing the check for no good reason.